src/airport.js

```diff
@@ -1,17 +1,21 @@
 const Plane = require(`./plane.js`);
-
 class Airport {
   // here's a starting point for you
   airportPlanes = [];
   capacity = 10;
 
   landPlane = plane => {
     if (plane instanceof Plane) this.airportPlanes.push(plane)
-  };
+    
+    };
 
   changeCapacity = capacity => {
-    this.capacity = capacity;
-    
-  }
+      this.capacity = capacity;
+  };
+
+  isAirportFull = capacity => {
```

Comment:
I think there is a better way to do this without making capacity = 10 and return true. Maybe using that the capacity is always higher than the length or the airportPlanes array but then maybe that defeats the object of a capacity exisiting?